cmd/validator/flags/flags.go

```diff
@@ -349,7 +349,6 @@ var (
 	EnableBuilderFlag = &cli.BoolFlag{
 		Name:  "enable-builder",
 		Usage: "Enables Builder validator registration APIs for the validator client to update settings such as fee recipient and gas limit. Note* this flag is not required if using proposer settings config file",
-		Value: false,
```

Comment:
Is this relevant?

Reply:
Yes, I found out that default values override configs, this is perhaps an issue through out the cli flags but in particular, I tested thsi one and realized that if you use enable-builder in the config file when it has a starting value (false) the config file does not override this value.